Common/src/main/java/com/blamejared/crafttweaker/impl/helper/AccessibleServerElementsProvider.java

```diff
@@ -0,0 +1,45 @@
+package com.blamejared.crafttweaker.impl.helper;
+
+import com.blamejared.crafttweaker.mixin.common.access.server.AccessReloadableServerResources;
+import com.blamejared.crafttweaker.platform.helper.IAccessibleServerElementsProvider;
+import com.google.common.base.Suppliers;
+import net.minecraft.server.ReloadableServerResources;
+
+import java.util.Objects;
+import java.util.function.Supplier;
+
+public final class AccessibleServerElementsProvider implements IAccessibleServerElementsProvider {
```

Comment:
Why not use `IAccessibleElementsProvider`?

Reply:
A client connected to the server won't have ServerResources since it isn't synced anywhere (afaik), would it not be a bit confusing being there when the `recipeManager` is always available?

I'm happy to merge them though, just let me know if you think that is the best idea